app/helpers/pipeline_runs_helper.rb

```diff
@@ -65,10 +65,15 @@ def aegea_batch_submit_command(base_command,
                                  docker_image: "idseq_dag",
                                  sample_id: 0,
                                  stage_name: "misc")
+    # TODO: environment values once we use the new pipeline infra
+    #   that new infra uses dev rather than development
+    #   once we switch to that for the pipeline these values won't be needed
+    deployment_environment = Rails.env == "development" ? "dev" : Rails.env
```

Comment:
There is a mapping for in https://github.com/chanzuckerberg/idseq-web/blob/master/app/services/sfn_pipeline_dispatch_service.rb#L13. Please use the mapping but feel free to move it a more adequate place if necessary.

Reply:
Since it is temporary, it is fine to use that location...